draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -804,6 +805,18 @@ when presenting to a downstream party.
 In some scenarios this behavior could be desirable,
 but if it is not, Issuers need to support and Verifiers need to enforce Holder Binding.
 
+## Explicit Typing {#explicit_typing}
+
+Section 3.11 of [@RFC8725] describes the use of explicit typing to prevent confusion attacks
+where one kind of JWT to be confused for another. SD-JWTs are also potentially
```

Comment:
```suggestion
Section 3.11 of [@RFC8725] describes the use of explicit typing to prevent confusion attacks
in which one kind of JWT is mistaken for another. SD-JWTs are also potentially
```